apps/evaluations/views/dataset_views.py

```diff
@@ -51,12 +51,10 @@ class DatasetTableView(SingleTableView, PermissionRequiredMixin):
     template_name = "table/single_table.html"
 
     def get_queryset(self):
-        from django.db.models import Count
-
         return (
             EvaluationDataset.objects.filter(team=self.request.team)
             .annotate(message_count=Count("messages"))
-            .order_by("name")
+            .order_by("-created_at")
```

Comment:
This will change how the datasets are ordered, but I think this ordering makes more sense?